packages/devtools/web/index.html

```diff
@@ -35,6 +35,7 @@
     <link rel="stylesheet" href="packages/devtools/src/logging/logging.css">
     <link rel="stylesheet" href="packages/devtools/src/performance/performance.css">
     <link rel="stylesheet" href="packages/devtools/src/profiler/profiler.css">
+    <link href="packages/devtools/src/settings/settings.css" rel="stylesheet">
```

Comment:
go ahead and update the other <link tags to include the `href` first. That is the better convention.

Reply:
done! for tags that doesn't only have rel and href tag (for example: size), currently I'm ordering it in lexicographical order, is this fine?
So the order is: href, rel, size

Reply:
Sounds fine. I'm not seeing your updated change to the index.html file though. perhaps you need to add it to your branch.